src/consensus/aft/raft.h

```diff
@@ -2236,14 +2296,17 @@ namespace aft
             !resharing_tracker->have_resharing_result_for(rr.value(), idx))
           {
             LOG_TRACE_FMT(
-              "Configurations: not switching to next configuration, resharing "
+              "Configurations: not switching to next configuration, "
+              "resharing "
```

Comment:
```suggestion
              "Configurations: not switching to next configuration, resharing "
```

Looks like a temporarily indented reflow that we can revert.